kong/plugins/http-log/schema.lua

```diff
@@ -40,8 +41,30 @@ return {
               referenceable = true,
             },
           }},
+          { queue = typedefs.queue },
```

Comment:
This feels like it needs to be added to (same with other plugins):
https://github.com/Kong/kong/blob/master/kong/clustering/compat/removed_fields.lua